packages/langium/src/parser/lexer.ts

```diff
@@ -4,11 +4,22 @@
  * terms of the MIT License, which is available in the project root.
  ******************************************************************************/
 
-import type { ILexingError, IMultiModeLexerDefinition, IToken, TokenType, TokenTypeDictionary, TokenVocabulary } from 'chevrotain';
+import type { ILexerErrorMessageProvider, ILexingError, IMultiModeLexerDefinition, IToken, TokenType, TokenTypeDictionary, TokenVocabulary } from 'chevrotain';
 import type { LangiumCoreServices } from '../services.js';
-import { Lexer as ChevrotainLexer } from 'chevrotain';
+import { Lexer as ChevrotainLexer, defaultLexerErrorProvider } from 'chevrotain';
 import type { LexingReport, TokenBuilder } from './token-builder.js';
 
+export abstract class AbstractLexerErrorMessageProvider implements ILexerErrorMessageProvider {
```

Comment:
Wouldn't it be more in line with the rest of the implementation (especially in the parser module) if this was instead:
```suggestion
export class DefaultLexerErrorMessageProvider implements ILexerErrorMessageProvider {
```
and then this was the class used in `src/default-module.ts` instead of using `defaultLexerErrorProvider` directly?